e2e-tests/production-runtime/package.json

```diff
@@ -5,8 +5,8 @@
   "author": "Kyle Mathews <mathews.kyle@gmail.com>",
   "dependencies": {
     "cypress": "3.4.1",
-    "gatsby": "^2.0.118",
-    "gatsby-plugin-manifest": "^2.0.17",
+    "gatsby": "2.17.5-dev-1572285531830",
+    "gatsby-plugin-manifest": "2.2.24-dev-1572285531830",
```

Comment:
ops ;) gatsby-dev side effect - we should revert this

Reply:
Oops :p, thanks! Removed it